go/vt/servenv/grpc_server.go

```diff
@@ -125,6 +131,18 @@ func createGRPCServer() {
 		opts = append(opts, grpc.KeepaliveParams(ka))
 	}
 
+	if *GRPCAuthPluginImpl != "" {
+		log.Infof("Auth plugin %v have been loaded", *GRPCAuthPluginImpl)
```

Comment:
Minor nit -- I'd change the log to "enabled auth plugin %v" and I would move this to the end of this block instead of the start of it.